pkg/components/render_test.go

```diff
@@ -151,8 +151,20 @@ func Test_renderTopolvmDaemonsetTemplate(t *testing.T) {
 		{
 			name: "renders lvmd-socket-name path",
 			args: args{
-				tb:   tb,
-				data: renderParamsFromConfig(config.NewMicroshiftConfig(), assets.RenderParams{"SocketName": "/run/lvmd/lvmd.socket", "lvmd": "foobar"}),
+				tb: tb,
+				data: renderParamsFromConfig(&config.MicroshiftConfig{
+					LogVLevel:       0,
```

Comment:
Why do we need to fill this additional info here (Test_renderTopolvmDaemonsetTemplate) ?